doc/make.py

```diff
@@ -39,7 +39,7 @@ class DocBuilder:
 
     def __init__(
         self,
-        num_jobs=0,
+        num_jobs="auto",
```

Comment:
Is `num_jobs` just a pandas thing? We use the `-j` flag for building the Cython / C extensions so wouldn't be opposed to just using that here for consistency as well, since it matches the Sphinx docs

Reply:
`num_jobs` was there before; I didn't mess with the name. I like that `num_jobs` is more descriptive, but certainly understand the argument for consistent…arguments 😉